data/profiles/taskrunner.py

```diff
@@ -1,52 +1,62 @@
-#!/usr/bin/python
-
 # Copyright 2016, EMC, Inc.
 
 """
 Download and execute python scripts from the RackHD tasks api in a loop
 """
 
-import requests
+import urllib2
```

Comment:
I believe this is going to break Brocade.  The last change to this file changed from urllib2 to requests to support Brocade (which broke Cisco support).  I think we are going to need some try/except logic here.